# German translations for gramps package.
# Copyright (C) 2013 THE gramps'S COPYRIGHT HOLDER
# This file is distributed under the same license as the gramps package.
#
# Mirko Leonhäuser <mirko@leonhaeuser.de>, 2013.
msgid ""
msgstr ""
"Project-Id-Version: gramps 34\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2013-09-29 13:12+0200\n"
"PO-Revision-Date: 2013-09-29 13:23+0200\n"
"Last-Translator: Mirko Leonhäuser <mirko@leonhaeuser.de>\n"
"Language-Team: German <kde-i18n-de@kde.org>\n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: Lokalize 1.5\n"

#: SourceReferences//SourceReferences.gpr.py:31
msgid "Source References"
msgstr "Ereignisreferenzen"

#: SourceReferences//SourceReferences.gpr.py:32
msgid "Gramplet showing the references for a source"
msgstr "Gramplet zeigt die Referenzen für eine Quelle"

#: SourceReferences//SourceReferences.gpr.py:39
msgid "References"
msgstr "Referenzen"

#: SourceReferences//SourceReferences.py:44
msgid "Type"
msgstr "Art"

#: SourceReferences//SourceReferences.py:45
msgid "Name"
msgstr "Name"

#: SourceReferences//SourceReferences.py:135
msgid ""
"Cannot open new citation editor at this time. Either the citation is already "
"being edited, or the associated source is already being edited, and opening "
"a citation editor (which also allows the source to be edited), would create "
"ambiguity by opening two editors on the same source. \n"
"\n"
"To edit the citation, close the source editor and open an editor for the "
"citation alone"
msgstr ""
"Kann im Moment keinen neuen Fundstelleneditor öffnen. Entweder wird die "
"Fundstelle oder die dazugehörige Quelle bereits bearbeitet und das Öffnen "
"eines Fundstelleneditors (der auch ermöglicht die Quelle zu bearbeiten), "
"erzeugt Unklarheit durch öffnen von zwei Editoren für die selbe Quelle.\n"
"\n"
"Um die Fundstelle zu bearbeiten schließe den Quelleneditor und öffne nur den "
"Fundstelleneditor"

#: SourceReferences//SourceReferences.py:148
msgid "Cannot open new citation editor"
msgstr "Kann neuen Fundstelleneditor nicht öffnen"

